added materialize library

chrisjmckeown opened the pull request:
Added the materialize library with header, footer and columns, to get the map in the correct location

chrisjmckeown committed:
added materialize library

smcheah commented on index.html:
```diff
@@ -16,51 +16,88 @@
+            <section class="col s12 m8 l9 main-body">
+                <div class="row">
+                    <div class="col s12 m6 l6">
+                        <div class="card bands-in-town-list">
+                            <div class="card-title">The bands in Town</div>
+                            <div class="card-content">
+                                <p></p>
+                            </div>
+                        </div>
+                        <div class="card bands-in-town-scroll">
+                            <div class="card-title">Possible carousel of bands</div>
+                            <div class="card-content">
+                                <p></p>
+                            </div>
+                        </div>
+                        <div id="blurb-about-site" class="card">
+                            <div class="card-title">Blurb about Site</div>
+                            <div class="card-content">
+                                <p> The content written will apear in here this is the dive here it goes in</p>
+                            </div>
+                        </div>
```
I think having the blurb card first is better (on top of the bandsintown list and scroll)

smoneo replied:
I think having the blurb card first is better (on top of the bandsintown list and scroll)
```suggestion
                        <div class="card" id="blurb-about-site">
                            <div class="card-title">Blurb about Site</div>
                            <div class="card-content">
                                <p> The content written will appear in here this is the dive here it goes in</p>
                            </div>
                        </div>
                        <div class="card">
                            <div class="card-title">The bands in Town</div>
                            <div class="card-content bands-in-town-list"></div>
                        </div>
                        <div class="card">
                            <div class="card-title">Possible carousel of bands</div>
                            <div class="card-content bands-in-town-scroll"></div>
                        </div>
```

smoneo commented on index.html:
```diff
@@ -16,51 +16,88 @@
+                                <p> The content written will apear in here this is the dive here it goes in</p>
+                            </div>
+                        </div>
+                    </div>
+                        <div class="col s12 m6 l6 map_canvas" id="map_canvas" >
+                    </div>
```
how do you feel about putting the map in a card?
```suggestion
                    <div class="col s12 m6 l6">
                        <div class="card">
                            <div class="map_canvas" id="map_canvas"></div>
                        </div>
                    </div>
```

chrisjmckeown committed:
Update index.html

Will have a look, was struggling to get the map to full the space when in a card, but will check again.

Co-authored-by: Simone Cheah <37390984+smcheah@users.noreply.github.com>

chrisjmckeown committed:
Update index.html

We had talked about making the website blurb hide when a search starts or is reset? can review tonight maybe

Co-authored-by: Simone Cheah <37390984+smcheah@users.noreply.github.com>

smoneo reviewed (approved):
> **nice**
